gptcli/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.2.0"
+__version__ = "0.2.1"
```

Comment:
could you please undo this change? i usually bump the version before making a release